app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_ages",
+    [
+        pytest.param(
+            0,
+            0,
+            [0, 0],
+            id="should return [0, 0] when inputs are equal to zeros"
+        ),
+        pytest.param(
+            14,
+            14,
+            [0, 0],
+            id="should return [0, 0] when inputs are "
+               "one less than 1 human year"
+        ),
+        pytest.param(
+            15,
+            15,
```

Comment:
also, consider to group test cases for 15-23 human years.

Reply:
Group them? I'm not sure I even have a single one.
Though it would make sense to add one to check if the behavior persists for larger inputs.